signalbot/storage.py

```diff
@@ -17,29 +22,28 @@ def save(self, key: str, object: Any):
 class StorageError(Exception):
     pass
 
+class SQLiteStorage(Storage):
+    def __init__(self, database = ":memory:"):
+        self._sqlite = sqlite3.connect(database)
+        self._sqlite.execute("CREATE TABLE IF NOT EXISTS signalbot (key text unique, value text)")
```

Comment:
The CI failed on linting on this file, please install `pre-commit` as described here and make sure all pre-commit checks are good https://github.com/filipre/signalbot?tab=readme-ov-file#local-development-and-package

Reply:
I installed and ran the pre-commit hooks, and while the whitespace checker caught the problems I see in the CI output, black passed and didn't complain about this formatting. I can apply the changes anyway but wanted to check first.